crates/sui-indexer-alt-jsonrpc/src/config.rs

```diff
@@ -37,6 +38,9 @@ pub struct RpcConfig {
 
     /// Configuring limits for the package resolver.
     pub package_resolver: sui_package_resolver::Limits,
+
+    /// Threshold for logging slow requests. Requests that take longer than this will be logged as warnings.
+    pub slow_request_threshold: Duration,
```

Comment:
Things may fit a bit better if you create a config struct and layer type to hold this, rather than having it bare in the top-level.